Corona-Warn-App/src/main/java/de/rki/coronawarnapp/worker/DiagnosisTestResultRetrievalPeriodicWorker.kt

```diff
@@ -54,7 +48,7 @@ class DiagnosisTestResultRetrievalPeriodicWorker @AssistedInject constructor(
         var result = Result.success()
         try {
 
-            if (abortConditionsMet()) {
+            if (abortConditionsMet(System.currentTimeMillis())) {
```

Comment:
We could inject `TimeStamper` into the worker as use it as time source, then we have full control over the time.

Reply:
Yes, will do